db/migrate/20150906182103_kill_profiles.rb

```diff
@@ -0,0 +1,9 @@
+class KillProfiles < ActiveRecord::Migration
+  def change
+    drop_table :profiles
+    drop_table :posts_profiles
+    [ :cafe_works, :candidates, :documents, :news, :rents ].each do |t|
```

Comment:
Detect this

![article_img](https://cloud.githubusercontent.com/assets/1269005/9720416/1f19c672-558e-11e5-97d6-4a1326ef949c.jpg)